lib/fixer.js

```diff
@@ -1,8 +1,112 @@
-module.exports = function fixer(parsedContent) {
-    // @todo if parsedContent.matches is empty, comment an error
-    // Return data if need to fix
+var crypto = require('crypto');
+
+function* reply(githubapi, content, message) {
+    if (content.type === 'pull_request_review_comment') {
+        yield githubapi.replyToPullRequestReviewComment({
+            repoUser: content.repository.user,
+            repoName: content.repository.name,
+            pullRequestNumber: content.pullRequest.number,
+            commentId: content.comment.id,
+            message: message,
+        });
+    }
+}
+
+function fixBlob(content, blob, treeBlob, match) {
```

Comment:
Can you give an example of input and output as comments? Or at least give more details in a comment? This function looks quite cryptic.